.github/workflows/flowzone.yml

```diff
@@ -0,0 +1,33 @@
+name: Flowzone
+
+on:
+  pull_request:
+    types: [opened, synchronize, closed]
+    branches:
+      - "main"
+      - "master"
+
+jobs:
+  flowzone:
+    name: Flowzone Linux
+    uses: product-os/flowzone/.github/workflows/flowzone.yml@ab77/operational
+    # FIXME: (re)inherit once SENTRY_AUTH_TOKEN secret standardised
+    #secrets: inherit
```

Comment:
Why can't you use `secrets: inherit` @ab77? It should work fine even with custom jobs, see https://github.com/balena-os/balena-supervisor/blob/v14.2.18/.github/workflows/flowzone.yml#L14 / https://github.com/balena-os/balena-supervisor/blob/v14.2.18/.github/actions/finalize/action.yml#L18 and lets you use sensible names in the custom jobs as well